pypeit/par/pypeitpar.py

```diff
@@ -703,6 +703,17 @@ def __init__(self, method=None, pixelflat_file=None, spec_samp_fine=None,
                                           'relative weights is given by ``nspec/slit_illum_smooth_npix``, ' \
                                           'where nspec is the number of spectral pixels.'
 
+        defaults['flatfield_structure'] = False
+        dtypes['flatfield_structure'] = bool
+        descr['flatfield_structure'] = 'Set this variable to True if you want to compute and ' \
+                                       'account for the detector structure in the flatfield image. ' \
+                                       'Note that ``detector structure`` refers to pixel sensitivity ' \
+                                       'variations that depend on both the spatial and spectral coordinates. ' \
+                                       'In most cases, the default 2D bspline is sufficient to account ' \
+                                       'for detector structure (i.e. set this parameter to False). Note ' \
```

Comment:
Can you please add a note to the docs indicating that currently this functionality is only used for KCWI.